app/views/eventRegistrations.html

```diff
@@ -42,20 +42,46 @@ <h2 class="page-title">Registrations</h2>
             <i class="fa fa-chevron-down" ng-show="!showMoreFilters"></i>
             <i class="fa fa-chevron-up" ng-show="showMoreFilters"></i>
           </button>
-          <button class="btn btn-primary" ng-click="refreshRegistrations()"><i class="fa fa-refresh"></i> Refresh</button>
-          <button type="button" class="btn btn-primary dropdown-toggle" dropdown-toggle>
-            Columns
-            <i class="fa fa-caret-down"></i>
+          <button class="btn btn-primary" ng-click="refreshRegistrations()">
+            <i class="fa fa-refresh"></i>
+            Refresh
           </button>
-          <ul class="dropdown-menu dropdown-menu-right reg-views-dropdown">
-            <li ng-repeat="block in blocks" ng-if="block.profileType !== 'NAME' && block.profileType !== 'EMAIL'">
-              <a href="" ng-click="toggleColumn($index)">
-                <i class="fa fa-check-square" ng-show="block.visible"></i>
-                <i class="fa fa-square-o" ng-show="!block.visible"></i>
-                {{block.exportFieldTitle || block.title}}
-              </a>
-            </li>
-          </ul>
+          <div class="btn-group" role="group" dropdown auto-close="outsideClick">
+            <button type="button" class="btn btn-primary dropdown-toggle" dropdown-toggle>
+              Display Columns
+              <i class="fa fa-caret-down"></i>
+            </button>
+            <ul class="dropdown-menu dropdown-menu-center reg-views-dropdown">
```

Comment:
Using `dropdown-menu-center` seems to push the dropdown off the page:
<img width="360" alt="screen shot 2016-07-12 at 11 43 30 pm" src="https://cloud.githubusercontent.com/assets/756501/16794091/7dd595fc-488a-11e6-8724-eecd8ab035a7.png">

I like the left align too but it doesn't fit on the page. Are you seeing the same thing?